rear_rider_device/leds_child_proc.py

```diff
@@ -113,6 +104,16 @@ async def on_led_strobe_off(self):
         self.writeline('on_led_strobe_off')
         await self.leds_proc.led_strobe_off()
     
+    async def on_discoverable(self):
+        """
+        The handler for when bluetooth changes its discoverability to other devices.
+        """
+        line = await self.readline()
+        if line == '1':
+            await self.leds_proc.set_discoverable_effect(True)
+        elif line == '0':
+            await self.leds_proc.set_discoverable_effect(False)
+    
```

Comment:
same thing here if `line` is only ever 1 or 0 

```suggestion
        line = await self.readline()
        await self.leds_proc.set_discoverable_effect(line)
```

Reply:
Same as above, line is a `string` not an `int` or `bool`.